### ISSUE TITLE
3.0.0	Error: Cannot find module

### ISSUE THREAD

**Issue by njzjz** [hexojs/hexo-util#290]
```
08:57:18.372 | internal/modules/cjs/loader.js:330
-- | --
08:57:18.372 | throw err;
08:57:18.372 | ^
08:57:18.373 |  
08:57:18.373 | Error: Cannot find module '/opt/buildhome/repo/node_modules/hexo-util/lib/index'. Please verify that the package.json has a valid "main" entry
08:57:18.373 | at tryPackage (internal/modules/cjs/loader.js:322:19)
08:57:18.373 | at Function.Module._findPath (internal/modules/cjs/loader.js:683:18)
08:57:18.373 | at Function.Module._resolveFilename (internal/modules/cjs/loader.js:953:27)
08:57:18.373 | at Function.Module._load (internal/modules/cjs/loader.js:842:27)
08:57:18.373 | at Module.require (internal/modules/cjs/loader.js:1026:19)
08:57:18.375 | at require (internal/modules/cjs/helpers.js:72:18)
08:57:18.375 | at Object.<anonymous> (/opt/buildhome/repo/node_modules/hexo-fs/lib/fs.js:8:26)
08:57:18.375 | at Module._compile (internal/modules/cjs/loader.js:1138:30)
08:57:18.375 | at Object.Module._extensions..js (internal/modules/cjs/loader.js:1158:10)
08:57:18.375 | at Module.load (internal/modules/cjs/loader.js:986:32)
08:57:18.376 | at Function.Module._load (internal/modules/cjs/loader.js:879:14)
08:57:18.376 | at Module.require (internal/modules/cjs/loader.js:1026:19)
08:57:18.376 | at require (internal/modules/cjs/helpers.js:72:18)
08:57:18.376 | at Object.<anonymous> (/opt/buildhome/repo/node_modules/hexo-cli/lib/find_pkg.js:4:22)
08:57:18.376 | at Module._compile (internal/modules/cjs/loader.js:1138:30)
08:57:18.376 | at Object.Module._extensions..js (internal/modules/cjs/loader.js:1158:10) {
08:57:18.377 | code: 'MODULE_NOT_FOUND',
08:57:18.377 | path: '/opt/buildhome/repo/node_modules/hexo-util/package.json',
08:57:18.377 | requestPath: 'hexo-util'
08:57:18.377 | }
```

**Comment by so1ve** [comment]
also met this error

**Comment by yoshinorin** [comment]
@njzjz @so1ve 
Thank you for your reporting & sorry so inconvenient.
Just now we released `v3.0.1` :)

> https://www.npmjs.com/package/hexo-util/v/3.0.1

Would you please check?

@stevenjoezhang 
Thank you very much for your fix :)
